tools/scripts-config-storybook-lib/.storybook-templates/main.js

```diff
@@ -79,24 +74,34 @@ const defaultMain = {
 				...config.plugins,
 				// use dynamic-cdn-webpack-plugin with default modules
 				new CDNPlugin({
-					exclude: Object.keys(getAllModules()).filter(name => name.match(/^(@talend\/|angular)/))
+					exclude: Object.keys(getAllModules()).filter(name => name.match(/^(@talend\/|angular)/)),
+                    disable: true, // temporaly disable the CDN pluggin, causing 404 on the cdn
```

Comment:
still valid?

Reply:
Yes and by the way I completely remove it as we now have only one storybook so this ok in terms of perf and size.
Here is why it do not works. Storybook as it's own inject mecanism which is not supported by our plugin.
![image](https://github.com/Talend/ui/assets/19857479/1cbafcaf-9d9d-43b0-8f7c-0efee328384d)